inc/admin/dashboard/namespace.php

```diff
@@ -59,7 +59,22 @@ function replace_root_dashboard_widgets() {
 	// Remove third-party widgets
 	remove_meta_box( 'dashboard_rediscache', 'dashboard', 'normal' );
 
+	global $wpdb;
 	$user = wp_get_current_user();
+
+	$invitations = $wpdb->get_var( $wpdb->prepare( "SELECT COUNT(1) FROM $wpdb->usermeta WHERE meta_key LIKE %s AND user_id = %d", 'new_user_%', $user->ID ) );
```

Comment:
This check happens in two places, perhaps we should create a new utility method?

Reply:
yes I am agree @fdalcin. Probably a method like `getNumberOfInvitations` to follow the DRY.